python/ray/package.py

```diff
@@ -0,0 +1,90 @@
+import hashlib
+from pathlib import Path
+from zipfile import ZipFile
+
+from enum import Enum
+from ray.experimental import internal_kv
+
+PKG_DIR='/tmp'
```

Comment:
```suggestion
PKG_DIR = "/tmp/ray/runtime_resources"
```

(please run scripts/format.sh to lint)